tests/tests-core/src/main/java/com/gentics/mesh/core/graphql/GraphQLEndpointStressTest.java

```diff
@@ -0,0 +1,91 @@
+package com.gentics.mesh.core.graphql;
+
+import java.io.IOException;
+import java.util.Collection;
+import java.util.List;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Consumer;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameters;
+
+import com.gentics.mesh.core.rest.graphql.GraphQLResponse;
+import com.gentics.mesh.rest.client.MeshRequest;
+import com.gentics.mesh.rest.client.MeshRestClient;
+import com.gentics.mesh.test.MeshTestSetting;
+import com.gentics.mesh.test.TestSize;
+
+import io.vertx.core.json.JsonObject;
+
+@RunWith(Parameterized.class)
+@MeshTestSetting(testSize = TestSize.FULL, startServer = true, inMemoryDB = true)
+public class GraphQLEndpointStressTest extends AbstractGraphQLEndpointTest {
+
+	protected final int simultaneousCalls;
+
+	protected MeshRestClient clientV2;
+	protected MeshRestClient clientV1;
+
+	@Parameters(name = "calls={0}")
+	public static Collection<Integer> paramData() {
+		int total = (int) streamQueries().count();
+		return List.of(1, 2, 3, 5, 8, 13, 21, total, 100, 250, 500, 1000);
```

Comment:
Run up to 1000 calls at once, to make Vert.x threads switch the native threads. Not always works :(